app/models/grda_warehouse/cas_project_client_calculator/boston.rb

```diff
@@ -170,14 +171,14 @@ def value_for_cas_project_client(client:, column:)
     private def default_shelter_agency_contacts(client)
       # TODO: this is being moved to custom pull from Clarity
 
-      # client.most_recent_pathways_or_rrh_assessment.
+      # client.most_recent_pathways_or_rrh_assessment_for_destination.
       #   question_matching_requirement('c_casemanager_contacts')&.AssessmentAnswer
```

Comment:
Will the contacts still be handled by this calculator class, or will it come from somewhere else?

Reply:
This still undecided, so I'm leaving it for now.  Probably here, but after import via custom imports.